method/application.js

```diff
@@ -0,0 +1,164 @@
+const Crut = require('ssb-crut')
+const pull = require('pull-stream')
+const paraMap = require('pull-paramap')
+const { isCloakedMsg: isGroup } = require('ssb-ref')
+const spec = require('../spec/application.js')
+
+module.exports = function Application (ssb) {
```

Comment:
I really like how you've merged all of the methods into this one file! Really good idea. Especially because of crut, the code is a lot less now so it fits into one file easily. I think we should carry this on else where! 👏